docs/users/fbc_workflow.md

```diff
@@ -0,0 +1,154 @@
+# FBC workflow
+
+The prerequisite for the File Based Catalog workflow is to convert the existing operator
+to FBC format. This can be done by following a [onboarding](./fbc_onboarding.md) documentation.
+
+After an existing operator is onboard or you introduce a brand new operator you can
+start the FBC workflow.
+
+## FBC templates
+File-based catalog templates serve as a simplified view of a catalog that can be updated
+by the user. The OPM currently supports 3 types of templates and it is up to the user which
+template the operator will be using.
+
+* Basic template
+* SemVer template
+* Composite template
+
+More information about each template can be found at [opm doc](https://olm.operatorframework.io/docs/reference/catalog-templates/).
+
+The recommended template from the maintainability point of view is `SemVer` or `Composite` with
+`SemVer`integration.
+
+## Generate catalogs using templates
+To generate a final catalog for an operator a user needs to execute different `opm`
+commands based on the template type. We as operator pipeline maintainers want
+to simplify this process and we prepared a `Makefile` with all pre-configured targets.
+
+To get the `Makefile` follow these steps
+
+```bash
+cd <operator-repo>/operator/<operator-name>
+wget https://raw.githubusercontent.com/redhat-openshift-ecosystem/operator-pipelines/main/fbc/Makefile
+```
+
+The right place for the Makefile is in the operator's root directory
+```
+.
+├── 0.0.1
+│   ├── manifests
+│   └── metadata
+├── catalogs.yaml
+├── catalog-templates
+├── ci.yaml
+├── composite-config.yaml
+└── Makefile
+
+```
+
+You can modify the Makefile based on your needs and use it to generate catalogs by running `make catalog`.
+The command uses the `opm` and converts templates into catalogs. The generated catalogs
+can be submitted as a PR in Github and once the PR is processed changes will be released to the
+OCP index.
+
+```bash
+$ tree catalogs
```

Comment:
I really got confused by this and assumed this was in the operator directory, but I see in the [other doc](https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/docs/users/fbc_onboarding.md#submit-fbc-changes) that this is in the repo root, which is great.

Reply:
yep, that directory is in repo root and is common for all bundles.